elastic/kibana
# [Infrastructure UI] Add unified search to hosts table

## Summary

Closes #139594 

This PR integrates the unified search component with the hosts table.

### How to test this PR
- Start a local ES instance
- Start a local metricbeat with `system` module and `add_host_metadata` processor enabled
- Enable `Infrastructure Hosts view` in Advanced Settings
- Go to Infrastructure > Hosts

It should be possible to:
- Save and Load queries
  - Saved queries should be usable on other pages that use the unified search (Discover, Lens...)
- Add/Remove Filters
- Filter by date
- Query using KQL or Lucene

### Screenshots
<img width="985" alt="image" src="https://user-images.githubusercontent.com/2767137/198039018-16126ab4-0eef-4656-a085-df56cbcbd654.png">

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/2767137/198039176-5eb2229b-dec5-4a2a-92c0-90e012661deb.png">
